css/src/components/breadcrumbs.scss

```diff
@@ -0,0 +1,51 @@
+$breadcrumb-font-size: $document-font-size !default;
+$breadcrumb-spacing-horizontal-start: 0.5em !default;
+$breadcrumb-spacing-horizontal-end: 0.25em !default;
+$breadcrumb-padding-vertical: 0.375em !default;
+$breadcrumb-separator: '\00200B/' !default;
+$breadcrumb-color: $hyperlink !default;
```

Comment:
I think it would be more flexible / separate concerns if breadcrumbs didn't set link colors.

Reply:
It seems pretty straightforward to me that breadcrumbs would ensure the link color.